hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/TestHoodieSparkSqlWriter.scala

```diff
@@ -591,8 +591,7 @@ class TestHoodieSparkSqlWriter {
         HoodieBootstrapConfig.PARALLELISM_VALUE.key -> "4",
         DataSourceWriteOptions.OPERATION.key -> DataSourceWriteOptions.BOOTSTRAP_OPERATION_OPT_VAL,
         DataSourceWriteOptions.RECORDKEY_FIELD.key -> "_row_key",
-        DataSourceWriteOptions.PARTITIONPATH_FIELD.key -> "partition",
-        HoodieBootstrapConfig.KEYGEN_CLASS_NAME.key -> classOf[NonpartitionedKeyGenerator].getCanonicalName)
+        DataSourceWriteOptions.PARTITIONPATH_FIELD.key -> "")
```

Comment:
shouldn't we set the key gen class instead of setting partition path config to empty string ?

Reply:
After this change, we have to set the partition path to empty String.  Otherwise, the following error is thrown:
```
org.apache.hudi.exception.HoodieException: Config conflict(key	current value	existing value):
PartitionPath:	partition
```

Reply:
Key gen class name config is kept.